### PR TITLE
add instagram icon

### LINKED ISSUE #147: Add instagram to social media section
The instagram icon is still missing, add it to [SocialMediaSection.tsx](https://github.com/OpenTechSchool/websitenext/blob/master/components/Section/SocialMediaSection.tsx) in the way the other icons are added.

The import:
`import InstagramIcon from '@material-ui/icons/Instagram';`

### PR DESCRIPTION
**Pull request by morrme**
Fixes #147

Note: The instagram.com/opentechschool IG account was blank, so I used instagram.com/opentechschool_leipzig . Please let me know if I need to change the address.

### CODE REVIEW

**Comment by morrme** [comment]
Here is the new icon:
preview of change

**Comment by morrme** [comment]
@gsambrotta @normade @xMartin I'd like a review please.

**Comment by xMartin** [comment]
I'd say the Leipzig Insta account should be linked from the Leipzig chapter page.

Not sure if we have a global account registered.

**Comment by morrme** [comment]
Thank you @xMartin ! Was the idea of the original issue (#147 ) based on the assumption that an account was already created? Or was it just to provide the icon without linking it anywhere?

**Comment by xMartin** [comment]
I don't know. @normade ?

Currently I'm not aware of anyone being available to create content for a global insta account.

**Comment by normade** [comment]
Sorry @morrme and @xMartin, the issue was not specific enough.
It was just meant for the Leipzig chapter page, as we run an Insta account for a while now and wanted to add it there.

**Comment by morrme** [comment]
@normade I see. So should I leave the icon in the import section, but remove it below where I linked to Leipzig's page?

**Comment by morrme** [comment]
@normade @xMartin  I have left the instagram icon but removed the chapter page link. Does this complete the issue?

**Comment by Twissi** [comment]
@morrme sorry for the confusion. We would like to add the insta account to the Leipzig city page only, not in the global social media section.

Here a more detailed description. Hope that helps:
1. Add the insta account to the markdown file for Leipzig (en/leipzig.md, de/leipzig), similar to facebook
2. Add the insta logo to the city page (as part of the `socialIconsList`) here: https://github.com/OpenTechSchool/websitenext/blob/master/pages/cities/%5Bslug%5D.js
3. Make sure the insta logo shows up at https://www.opentechschool.org/cities/leipzig and links to the Leipzig insta account
4. Make sure the insta logo does not show up at other city pages for example https://www.opentechschool.org/cities/berlin

**Comment by morrme** [comment]
@Twissi 
Here is the new Leipzig page social section: 
leipzig ig

And here is Berlin:

berlinnoig

File: components/Section/SocialMediaSection.tsx [hunk 1]
```diff
@@ -59,6 +61,16 @@ export default function SocialMediaSection({ bgColor = '' }) {
             <GitHubIcon style={iconStyle} />
           </a>
         </Grid>
+        <Grid item xs={4} md={1}>
```
Comment (Twissi) [inline components/Section/SocialMediaSection.tsx:64]: Instagram can be removed from the SocialMediaSection for now because we don't have a link yet

**Comment by morrme** [comment]
@twissi All done!

**Comment by Twissi** [comment]
@morrme looks great! Thank you so much!

**Pull request merged**